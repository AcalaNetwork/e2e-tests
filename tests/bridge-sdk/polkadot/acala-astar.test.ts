import { bridgeSDKTest } from '../shared'

bridgeSDKTest({
  from: 'acala',
  to: 'astar',
  token: 'AUSD',
})
