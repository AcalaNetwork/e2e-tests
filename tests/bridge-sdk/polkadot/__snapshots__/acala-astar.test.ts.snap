// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`'acala' to 'astar' using bridgeSDK cross-chain 'AUSD' > after 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": 8,
  "toChain": "(rounded 12)",
}
`;

exports[`'acala' to 'astar' using bridgeSDK cross-chain 'AUSD' > fee 1`] = `"(rounded 0.0003)"`;

exports[`'acala' to 'astar' using bridgeSDK cross-chain 'AUSD' > initial 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": 10,
  "toChain": 10,
}
`;
